third_party/bitcoin-core/BUILD.gn

```diff
@@ -36,12 +36,14 @@ static_library("bitcoin-core") {
   sources = [
     "src/src/base58.cpp",
     "src/src/base58.h",
+    "src/src/bech32.cpp",
+    "src/src/bech32.h",
     "src/src/crypto/ripemd160.cpp",
     "src/src/crypto/ripemd160.h",
+    "src/src/util/strencodings.h",
   ]
 
   sources += [
-    "//brave/chromium_src/brave/third_party/bitcoin-core/src/src/serialize.h",
```

Comment:
Shouldn't there be a build error complaining about use of exception? Which is why we added [this](https://github.com/brave/brave-core/pull/8243/commits/261db3836286af02200cce70481b89e9f85a50a6) to deal with it without specifying `//build/config/compiler:exceptions`

Reply:
Well, that passed CI and my local compilation. But I think I completely misinterpreted what was going on here, so reverted changes to `serialize.h`